js/chips.js

```diff
@@ -40,35 +70,42 @@ function validateMoves(currentSquare) {
 	// - if a capture is available, create an array of valid squares to *jump-to* that would *capture* an enemy chip
 }
 
-function evalSquare(selectedSquare) {
+function evalSquare() {
 
-	// if the clicked square is the parent of the selectedChip, do nothing.
-	if (selectedSquare.children[0] == selectedChip) {
-		return false; 
-	}
+	//only run this function if a square is selected. Will not work if global var selectedSquare is undefined
```

Comment:
Had to add if-statement, so that the function will only run if a square has been selected. Otherwise will get error messages of "cannot check property of undefined" in the following steps. But I think this is a silly fix - why even call the function evalSquare if a square is not selected? Long story short, I couldn't make it work another way _yet_, so I'll have to figure it out soon.